docs/env.md

```diff
@@ -4,7 +4,7 @@ Here we describe all environment variables that can be set for the docs applicat
 
 ## impress-backend container
 
-These are the environmental variables you can set for the impress-backend container.
+These are the environment variables you can set to build the `impress-backend` container.
```

Comment:
These one are not here to build the container but to change the settings at runtime. I apologize my comment was too general but the `to build` should be reverted